### ISSUE TITLE
Little Tiled Water has no FOV decrease, blue fog, and has some rendering problems.

### ISSUE THREAD

**Issue by SealTheTiel** [CreativeMD/LittleTiles#405]
### Issue
Title should explain it all but I'll explain more.

![image](https://user-images.githubusercontent.com/38400820/82048608-8b7c1c80-96e7-11ea-8e93-b1f6facc9953.png)

When your head is in a vanilla water block, it has this blue fog(which decreases with respiration enchant, optifine) and your FOV decreases as if you have been affected by Slowness
![image](https://user-images.githubusercontent.com/38400820/82048631-93d45780-96e7-11ea-9df8-fd2c9f9c5a50.png)

However, in little tiled water, there are no effects applied to the screen and the water itself isn't rendered(physics work fine tho).
![image](https://user-images.githubusercontent.com/38400820/82048650-99ca3880-96e7-11ea-9025-d7b1887793c1.png)

(will now start including this in my future reports)
Little Tiles Version : LittleTiles_v1.5.0-pre199_28_mc1.12.2
Forge Version : 14.23.5.2854
Optifine Version : 1.12.2 HD U F6 Pre1

**Comment by kirokadura** [comment]
The LT water block (both the normal and white variety) is a block that merely simulate certain properties of water, such as slowness, texture and flow. 

It does apply a filter to the screen however, albeit a very light one. 

![2020-05-15_17 59 02](https://user-images.githubusercontent.com/29471180/82095714-fe7c9780-96d5-11ea-838c-99b1d26a3771.png)
![2020-05-15_17 59 06](https://user-images.githubusercontent.com/29471180/82095717-ffadc480-96d5-11ea-9881-af96e3d5712f.png)
![2020-05-15_17 59 08](https://user-images.githubusercontent.com/29471180/82095720-00465b00-96d6-11ea-946c-222f9ce4fe6d.png)
![2020-05-15_17 59 10](https://user-images.githubusercontent.com/29471180/82095723-00def180-96d6-11ea-882d-491fde499843.png)

Might be that the new Optifine F6 breaks this feature. I'll have to test it when I get the chance.

**Comment by SealTheTiel** [comment]
Ah, I just tested the colored water blocks and they do have their respective colors' tint, though it is on the render distance fog so it should be working as intended, I just haven't tested it enough.
FOV is also the same either way but I think it's alright.
During testing, the opaque water block seems to have this same issue as #109 
![image](https://user-images.githubusercontent.com/38400820/82115008-c5eec380-9792-11ea-9928-734c61ee05aa.png)
![image](https://user-images.githubusercontent.com/38400820/82115009-c6875a00-9792-11ea-8d54-5b9e01ae8ea3.png)
I have removed Optifine, FoamFix and Phosphor in ^ pictures.
MipMap Levels 1 and 4 had no effect on that.

All that said, I have a few suggestions:
[1] Make the Little tile water to be more water like (render the water inside the block)
[2] Make the tint/fog more obvious
